submissions/victorAienobe/database/init.sql

```diff
@@ -0,0 +1,128 @@
+-- Created database for my e-commerce store
+CREATE DATABASE e_commerce_store;
+USE e_commerce_store;
+
+-- Created the categories table
+CREATE TABLE categories (
+	id INT NOT NULL AUTO_INCREMENT,
+	category_name VARCHAR(20) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+-- Created the admins table
+CREATE TABLE admins (
+	id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(50) NOT NULL,
+	email VARCHAR(50) NOT NULL,
+	password VARCHAR(15) NOT NULL,
+	phone_number VARCHAR(15) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+-- created the products table
+CREATE TABLE products (
+	id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(50) NOT NULL,
+	description VARCHAR(250) NOT NULL,
+	image BLOB,
+    category_id INT NOT NULL,
+    admin_id INT NOT NULL,
+	unit_price DECIMAL(4,0) NOT NULL,
+    stock_level INT NOT NULL,
+    status VARCHAR(20) NOT NULL,
+	PRIMARY KEY (id),
+    FOREIGN KEY (category_id) REFERENCES categories (id) ON DELETE CASCADE,
+    FOREIGN KEY (admin_id) REFERENCES admins (id) ON DELETE CASCADE
+);
+drop table products;
+-- Created the customers table
+CREATE TABLE customers (
+	id INT NOT NULL AUTO_INCREMENT,
+	first_name VARCHAR(20) NOT NULL,
+	last_name VARCHAR(20) NOT NULL,
+	email_address VARCHAR(40) NOT NULL,
+	password VARCHAR(15) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+-- Created the customer addresses table
+CREATE TABLE customer_addresses (
+	id INT NOT NULL AUTO_INCREMENT,
+	customer_id INT NULL,
+	street_address VARCHAR(100) NOT NULL,
+	city VARCHAR(20) NOT NULL,
+	state VARCHAR(20) NOT NULL,
+	zip_code INT,
+	country VARCHAR(20) NOT NULL,
+	phone_number BIGINT NOT NULL,
+    PRIMARY KEY (id),
+	FOREIGN KEY (customer_id) REFERENCES customers (id) ON DELETE CASCADE
+);
+
+-- Created the orders table
+CREATE TABLE orders (
+	id INT NOT NULL AUTO_INCREMENT,
+	customer_orders INT NOT NULL,
+	order_amount DECIMAL(7,2) NOT NULL,
+	order_created_at DATETIME,
+	PRIMARY KEY (id),
+	FOREIGN KEY (customer_orders) REFERENCES customers (id) ON DELETE CASCADE
+);
+
+-- Created the order items table
+CREATE TABLE order_items (
+	id INT NOT NULL AUTO_INCREMENT,
+	order_id INT NOT NULL,
+	product_id INT NOT NULL,
+	quantity INT NOT NULL,
+	unit_price DECIMAL(4,0) NOT NULL,
+	total_amount DECIMAL(10,2) NOT NULL,
+    PRIMARY KEY (id),
+	FOREIGN KEY (order_id) REFERENCES orders (id) ON DELETE CASCADE,
+	FOREIGN KEY (product_id) REFERENCES products (id) ON DELETE CASCADE
+);
+
+-- Inserted values into the categories table
+INSERT INTO categories (category_name)
+VALUES ('game console'),
+('Mens-shirt'),
+('soccer boots'),
+('club jerseys');
+
+-- Inserted values into the admins table
+INSERT INTO admins (name, email, password, phone_number) 
+VALUES ('victor aienobe', 'viktornobe@gmail.com', 'smoke&mirrors', +2348108088144),
```

Comment:
phone_number should be wrapped in quotes. The `+` sign in front of it should make it be a string and not  INT

Reply:
thanks.. corrections have been made